From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@alexandair : Thanks for your contribution to the Azure documentation! The author, @cynthn, has been notified to review your proposed change.

cynthn commented:
#sign-off @alexandair thank you for fixing that!

carolinacmoravia committed:
M24916: Missing preposition: "to"

Hello, @msfussell, 

Translator has reported possible source content issue.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

kevinvngo committed:
Clarified geo-restore

billmath committed:
updating gdpr

billmath committed:
updating

billmath committed:
updating

Commit:
toc: changed Overview to a dropdown with 1 item "About...".

Overview: changed "Face API" to "About Face API".
Fixed a few Acrolinx grammar warnings.

toc and index: changed "API Reference" to "Face API".
Added new SDKs per API Reference Guidance.

toc: added stock Resources items.

APIReference: Deleted obsolete link to Dev Portal
(prev redirected to Face Algorithm APIs
 which is already linked to in doc).

Commit:
Moved python below node.js.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FaceAPI

jmprieur committed:
New article for AAD V2 samples

jmprieur committed:
fixing broken link to other article

jmprieur committed:
Shortening level 2 titles (for in doc toc)

msvijayn committed:
add troubleshooting for another error

v-deken committed:
Updated Bing Web Search landing page to meet MVC standards

v-deken committed:
Fixed card issue in Resources section

msvijayn committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into vijayn-docs

msvijayn committed:
MASTER correct

jmprieur committed:
Fixing a typo and improvin the readability of a title

mattbriggs committed:
update paramater values

billmath committed:
updating

anil0225 committed:
redirection

anil0225 committed:
Master Merge

anil0225 committed:
Puting redirection Back

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into gdpr

billmath committed:
updating

billmath committed:
fixing aadchealth

mmacy committed:
[ACR] add Build to landing page

mmacy committed:
[ACR] tutorial title

aljo-microsoft committed:
Update service-fabric-common-questions.md

Adding FAQs for todays published Common Name documentation

aljo-microsoft committed:
Update service-fabric-common-questions.md

Removed extra space

aljo-microsoft committed:
Update service-fabric-common-questions.md

Using Acrolinx suggested Resource Manager vs ARM acronim

douglaslMS committed:
Removed author no longer at Microsoft.

rboucher committed:
adding note about multi dimensional metrics.

CelesteDG committed:
Voice and tone and general cleanup updates

Went through the doc and updated it for voice and tone (including sentence case for topic titles, some simplification, and removing a couple of redundant texts).

anil0225 committed:
Merge branch 'master' of https://github.com/anil0225/azure-docs-pr into cornerstoneredirection

snehithm committed:
Added ARM template for metric alerts

Saisang committed:
remove loc info in links

snehithm committed:
Minor changes

CelesteDG committed:
Updated for voice and tone and consistency

Updated to align with proposal on the v2 endpoint code samples topic (voice and tone, some simplification, and consistency tweaks).

PRMerger15 committed:
Merge pull request #39235 from aljo-microsoft/patch-3

Update service-fabric-common-questions.md

mtillman committed:
password sync --> password hash sync

GitHubber17 committed:
Merge pull request #39229 from mmacy/acr-landing-acrbuild

[ACR] add Build to landing page

TrishaMc committed:
Merge pull request #39238 from douglaslMS/adf-shengc-batch1

Removed author no longer at Microsoft.

Commit:
convert TOC.md to yml

snehithm committed:
Updating the TOC file

daveba committed:
still working on breadcrumb file

rmca14 committed:
Merge pull request #39148 from v-deken/LandingPage-WebSearch

Updated Bing Web Search landing page to meet MVC standards

GitHubber17 committed:
Merge pull request #39251 from Saisang/check-loc-info-removed

remove loc info in links

anil0225 committed:
Updating Redirection

GitHubber17 committed:
Merge pull request #39258 from daveba/contextual-breadcrumb

[Azure AD MSI] Still working on breadcrumb file

v-stadam committed:
change Search to Spell Check

v-stadam committed:
remove link redundancy

v-stadam committed:
standardize/correct TOCs

v-stadam committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into TOCfixes

anil0225 committed:
Missing /active-directory/

PRMerger18 committed:
Merge pull request #39174 from mattbriggs/mdb_20180426_pki2

update parameter values

areddish committed:
Update to pass named args

v-stadam committed:
shorten TOC entry, remove comments

roygara committed:
Updating other two ToC's with azure metrics change.

v-stadam committed:
add brand to title metadata

asmalser-msft committed:
Updating prerequisites

asmalser-msft committed:
Update active-directory-saas-salesforce-provisioning-tutorial.md

PRMerger15 committed:
Merge pull request #39267 from areddish/update-python-args

Update to pass named args

snehithm committed:
Fixing review comments

Added Azure to the title

PRMerger18 committed:
Merge pull request #39270 from asmalser-msft/patch-6

Updating prerequisites

snehithm committed:
Fixing comments for classic metric alerts ARM docs

Added Azure to title

Commit:
Made changes from cjgronlund <notifications@github.com>.

redirection.json: added redirect from index.md to Overview.
cognitive-services/index.yml: made main CS page link to Overview.

Deleted landing page (index).
Overview: Changed "About " to "What is ".

toc: deleted index.
Renamed Overview sub-item.
Renamed Quickstarts to "Try Face API" and moved to "How to".
Renamed Tutorials to "First Face API apps" and moved to "How to".

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FaceAPI

GitHubber17 committed:
Merge pull request #39205 from anil0225/cornerstoneredirection

Cornerstone Redirection

GitHubber17 committed:
Merge pull request #39250 from snehithm/metric-alerts-arm-templates

Added ARM template for metric alerts

GitHubber17 committed:
Merge pull request #39253 from mtillman/mtillman426mstr

Azure AD DS: password sync --> password hash sync

GitHubber17 committed:
Merge pull request #39269 from roygara/ToCchanges

Updating other two ToC's with azure metrics change.

Commit:
Update storage-use-azcopy-linux.md

PRMerger14 committed:
Merge pull request #39274 from seguler/patch-2

Update storage-use-azcopy-linux.md

jmprieur committed:
Merge pull request #2 from CelesteDG/patch-6

Voice and tone and general cleanup updates

jmprieur committed:
Merge pull request #3 from CelesteDG/patch-7

Updated for voice and tone and consistency

msvijayn committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into vijayn-docs

msvijayn committed:
MASTER check

msvijayn committed:
MASTER sync

PRMerger14 committed:
Merge pull request #39240 from rboucher/patch-8

adding note about multi dimensional metrics.

billmath committed:
updating

billmath committed:
change wording

PRMerger16 committed:
Merge pull request #39297 from billmath/compat

Fixing wording

PRMerger18 committed:
Merge pull request #37355 from carolinacmoravia/patch-668

M24916: Missing preposition: "to"

PRMerger18 committed:
Merge pull request #39026 from kevinvngo/patch-26

Clarified geo-restore

itechedit committed:
Merge pull request #39032 from billmath/gdpr

updating gdpr

GitHubber17 committed:
Merge pull request #39141 from jmprieur/code-samples-v2

New article for AAD V2 samples

jaredr80 committed:
Move Australiacentral/2 to Australia govt

jaredr80 committed:
Move Australiacentral/2 to Australia govt

jaredr80 committed:
Move Australia Central/2 to Govt

barclayn committed:
adding the link to the key vault pricing page and verifying that the other submissions sent by the customer were already part of the article

GitHubber17 committed:
Merge pull request #39257 from v-savila/upd1247150

convert TOC.md to yml

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger14 committed:
Merge pull request #39305 from jaredr80/patch-13

Move Australiacentral/2 to Australia govt

PRMerger14 committed:
Merge pull request #39307 from jaredr80/patch-16

Move Australia Central/2 to Govt

PRMerger17 committed:
Merge pull request #39308 from barclayn/disk-encryption

adding the link to the key vault pricing page and verifying that the …

v-alje committed:
Merge pull request #39311 from MicrosoftDocs/FromPublicRepo

From public repo

GitHubber17 committed:
Merge pull request #39286 from msvijayn/vijayn-docs

Adding new error message

CarlRabeler committed:
Update sql-database-vcore-resource-limits.md

PRMerger17 committed:
Merge pull request #39306 from jaredr80/patch-14

Move Australiacentral/2 to Australia govt

itechedit committed:
Merge pull request #39115 from easyj2j/FaceAPI

Face API

PRMerger20 committed:
Merge pull request #39314 from CarlRabeler/patch-55

Update sql-database-vcore-resource-limits.md

ktoliver committed:
Merge pull request #39265 from v-stadam/TOCfixes

Standardize TOCs and related changes

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
